include/GafferScene/EditScopeAlgo.h

```diff
@@ -162,6 +162,8 @@ GAFFERSCENE_API const Gaffer::GraphComponent *renderPassOptionEditReadOnlyReason
 
 GAFFERSCENE_API const Gaffer::GraphComponent *renderPassesReadOnlyReason( const Gaffer::EditScope *scope );
 
+GAFFERSCENE_API void renameRenderPass( Gaffer::EditScope *scope, const std::string &oldName, const std::string &newName );
```

Comment:
I wonder if there's any value in returns true/false to indicate whether anything happened or not? I'm thinking about someone using this to attempt renaming throughout a graph - might be useful to be able to give a summary of what was affected afterwards?

Reply:
Great, I like the idea. As mentioned above it's part of 8edae92.